app/services/gtfs_graph.rb

```diff
@@ -58,24 +66,24 @@ def ssp_schedule_async
   end
 
   def ssp_perform_async(trip_ids, agency_map, route_map, stop_map)
-    log "Load GTFS"
+    log 'Load GTFS'
     @gtfs.agencies
     @gtfs.routes
     @gtfs.stops
     @gtfs.trips
     load_gtfs_id_map(agency_map, route_map, stop_map)
     trips = trip_ids.map { |trip_id| @gtfs.trip(trip_id) }
-    log "Create changeset"
-    changeset = Changeset.create()
-    log "Create: SSPs"
+    log 'Create changeset'
+    changeset = Changeset.create!
+    log 'Create: SSPs'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.